renku/cli/init.py

```diff
@@ -75,15 +75,41 @@
     https://github.com/SwissDataScienceCenter/renku-project-template@master
     ... OK
 
-    INDEX  ID              DESCRIPTION
-    -----  --------------  -------------------------------------------------
-    1  python-minimal  Basic Python Project: The simplest Python-based[...]
-    2  R-minimal       Basic R Project: The simplest R-based renku proj[...]
+    INDEX ID             DESCRIPTION                VARIABLES
+    ----- -------------- -------------------------- ----------------------
+    1     python-minimal Basic Python Project:[...] description: proj[...]
+    2     R-minimal      Basic R Project: The [...] description: proj[...]
 
     Please choose a template by typing the index:
 
-Updating an existing project
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provide variables
+~~~~~~~~~~~~~~~~~
+
+Some templates require variables to properly initialize a new project. You
+can check them by listing the templates `--list-templates`.
+
+To provide variables, use the ``--variables```option and list them as in a
+Python dictionary or a JSON object requiring either single quotes ``'`` or
+single quotes ``"`` for variables names.
+
+.. code-block:: console
+
+    $ renku init --template-id python-minimal --variables \
+    '{ "description": "my new shiny project" }'
```

Comment:
could the ux around this be a bit nicer, maybe like this?

```
renku init --template-id python-minimal --parameter description="my new shiny project" --parameter another-parameter=value
```

This would follow the click convention for specifying multiple parameters - forcing users to write JSON might not go over so well :)

Reply:
I definitely like this idea :+1: